src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ReviewHearingRequirementsPreparer.java

```diff
@@ -49,7 +49,13 @@ public boolean canHandle(
 
         final YesOrNo reviewedHearingRequirements =
             asylumCase.read(AsylumCaseFieldDefinition.REVIEWED_HEARING_REQUIREMENTS, YesOrNo.class)
-            .orElseThrow(() -> new IllegalStateException("reviewHearingRequirements flag should be present"));
+                .orElse(null);
```

Comment:
I think this would be better returning `Optional.empty()` rather than a null.  Better to avoid null values where possible.